apis/management.cattle.io/v3/authn_types.go

```diff
@@ -216,6 +216,7 @@ type LdapConfig struct {
 	GroupMemberUserAttribute        string   `json:"groupMemberUserAttribute,omitempty"    norman:"default=entryDN,notnullable"`
 	GroupMemberMappingAttribute     string   `json:"groupMemberMappingAttribute,omitempty" norman:"default=member,notnullable,required"`
 	ConnectionTimeout               int64    `json:"connectionTimeout,omitempty"           norman:"default=1000,notnullable,required"`
+	NestedGroupMembershipEnabled    *bool    `json:"nestedGroupMembershipEnabled,omitempty"    norman:"default=true"`
```

Comment:
this does not need to be a pointer and we should avoid pointers where possible. AD's needs to be a pointer bc of backwards compatibility issues (bc it was already released)